dvc/command/root.py

```diff
@@ -17,7 +17,7 @@ def run(self):
 
 
 def add_parser(subparsers, parent_parser):
-    ROOT_HELP = "Relative path to the repository's directory."
+    ROOT_HELP = "Return the relative path to the DVC project."
```

Comment:
```suggestion
    ROOT_HELP = "Return the relative path to the DVC project's root."
```

Reply:
Hmmm I thought it's obvious but I guess you're right, since you may be using `dvc root` from inside the project (probably even the main/only use case!) Updated, thanks.